suites/pacific/cephadm/tier-1_5-1_service-apply-spec.yaml

```diff
@@ -66,6 +67,12 @@ tests:
           custom_image: true
           mon-ip: node1
           fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+          config:
```

Comment:
I think it would be if we keep out this ceph config out of args. would this make sense,
```
- test: 
    config:
        cmd: bootstrap
        base_cmd_args: 
             verbose: true
        args:
             mon-ip: node1
        ceph_config:
             container_image: <image>
```

Reply:
Sure we can.

Reply:
@sunilkumarn417 
Since it is one of the arguments we are passing to bootstrap command, I guess it would still make sense to keep it within args. what do you think?

Reply:
Got it.
``` 
  --config CONFIG, -c CONFIG
                        ceph conf file to incorporate
 ```